### DISCUSSION TITLE
Welcome to Oshun Discussions!

### DISCUSSION THREAD

**Discussion by Ovid** [Perl-Oshun/oshun#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * But first, [read the FAQ](https://github.com/Ovid/data-checks/discussions/3).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rwp0** [comment]
I'm a Perl user with Technical Support and System Administration experience that would very much like to contribute to Perl's future, growth, and adoption.

Currently I read posts in this community, and interact where I can.

---

I wish Perl/perl5 GH repositony also had Discussions enabled where the Perl community (ie. language users) could discuss anything (not strictly technical) relating to Perl as well.
